pandas/core/indexes/base.py

```diff
@@ -1110,19 +1110,37 @@ def to_series(self, index=None, name=None):
         return Series(self._to_embed(), index=index, name=name)
 
     def to_frame(self, index=True):
-        """
-        Create a DataFrame with a column containing the Index.
+        """Create a DataFrame with a column containing the Index.
 
         .. versionadded:: 0.21.0
 
         Parameters
         ----------
-        index : boolean, default True
-            Set the index of the returned DataFrame as the original Index.
+        index : boolean
```

Comment:
@datapythonista @jorisvandenbossche defaults on the same line, yes?

Reply:
Yes, the documentation still says to use it the numpy way (in brackets after the description), but in pandas-dev the consensus seems to be keeping the pandas way, which was already right in this case. Will update the documentation asap and send an email with the last minute changes to the organizers

Reply:
Yes, the docstring guide has been updated, but apparently only in the source: https://github.com/python-sprints/python-sprints.github.io/blob/master/pandas/guide/source/pandas_docstring.rst and not yet in the online version